### PR TITLE
Use tags rather than time-based logs for information on requests

### PR DESCRIPTION
**Pull request by NyanKiyoshi**
The information we were passing we not time-based.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [x] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/graphql/views.py [hunk 1]
```diff
@@ -218,13 +216,10 @@ def execute_graphql_request(self, request: HttpRequest, data: dict):
...
-                    {
-                        "query": document.document_string[
-                            : settings.OPENTRACING_MAX_QUERY_LENGTH_LOG
-                        ]
-                    }
-                )
+                raw_query_string = document.document_string[
+                    : settings.OPENTRACING_MAX_QUERY_LENGTH_LOG
+                ]
+                span.set_tag("GraphQL.query", raw_query_string)
```
Comment (patrys) [inline saleor/graphql/views.py:222]: Other places use lowercase names: `graphql.parent_type`, `graphql.field_name`.

Comment (NyanKiyoshi) [reply saleor/graphql/views.py]: Changed then

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 All notable, unreleased changes to this project will be documented in this file. For the released changes, please visit the [Releases](https://github.com/mirumee/saleor/releases) page.
 
 ## [Unreleased]
+- Use tags rather than time-based logs for information on requests - #5608 by @NyanKiyoshi
```
Comment (patrys) [inline CHANGELOG.md:6]: Not sure we need to mention this as this will likely ship as part of 2.10

File: saleor/graphql/views.py [hunk 3]
```diff
@@ -209,7 +207,7 @@ def parse_query(
     def execute_graphql_request(self, request: HttpRequest, data: dict):
         with opentracing.global_tracer().start_active_span("graphql_query") as scope:
             span = scope.span
-            span.set_tag(opentracing.tags.COMPONENT, "GraphQL")
+            span.set_tag(opentracing.tags.COMPONENT, "graphql")
```
Comment (patrys) [inline saleor/graphql/views.py:210]: Why change this?

Comment (NyanKiyoshi) [reply saleor/graphql/views.py:210]: I would expect the component having the same name as used in tags

File: CHANGELOG.md (hunk 2 above)
Comment (NyanKiyoshi) [reply CHANGELOG.md:6]: This should be part of the changelogs of rc3

File: saleor/graphql/views.py (hunk 3 above)
Comment (patrys) [reply saleor/graphql/views.py:210]: They are unrelated though?

**Pull request merged**